zdx/zdx.go

```diff
@@ -1,23 +1,14 @@
-// Package zdx provides an API for creating, merging, indexing, and querying zng
-// index files, a la SST in LSM trees, where a zdx "bundle" comprises a sequence of
-// key,value zng records and the records are sorted by key.
+// Package zdx provides an API for creating, merging, indexing, and querying
+// microindexes.
 //
-// A table on disk consists of the base table with zero or more b-tree files.
-// These files are all named with the same path prefix, e.g., "zdx.zng", where the
-// base table is zdx.zng and the b-tree index files, if any, are zdx.1.zng,
-// zdx.2.zng, and so forth.  The index files can always be reconstructed from
-// the base table.
+// A microindex comprises a base index section followed by zero or more parent
+// section indexes followed by a trailer.  The sections are organized into a
+// btree-like data structure so keys can be looked up efficiently without
```

Comment:
Nit: This name is usually styled as B-tree.
```suggestion
// B-tree-like data structure so keys can be looked up efficiently without
```